test/Microsoft.IdentityModel.Tokens.Tests/ECDsaSecurityKeyTests.cs

```diff
@@ -61,5 +61,14 @@ public void Defaults()
         {
             // there are no defaults.
         }
+
+        [Fact(DisplayName = "EcdsaSecurityKeyTests: IsSupportedAlgorithm")]
```

Comment:
I stopped setting DisplayName as it makes it harder to find the fully qualified method when debugging.
We should use either TheoryData or Inline so that all test cases will get executed. This test could fail on first one, then the others will not get run.